pkg/policy/selectorcache.go

```diff
@@ -330,9 +532,17 @@ func AddIdentitySelector(user CachedSelectionUser, selector api.EndpointSelector
 	return selectorCache.AddIdentitySelector(user, selector)
 }
 
-// RemoveIdentitySelector removes CachedSelector for the user.
+// RemoveSelector removes CachedSelector for the user.
 func RemoveIdentitySelector(user CachedSelectionUser, selector CachedSelector) {
-	selectorCache.RemoveIdentitySelector(user, selector)
+	selectorCache.RemoveSelector(user, selector)
+}
+
+func UpdateFQDNSelector(fqdnKey string, identities []identity.NumericIdentity) {
```

Comment:
Create a function that receives multiple FQDNKeys and a slice of identities to do [2]

Reply:
Can you clarify what the advantage of this is? Less locking on the SelectorCache?

Reply:
@ianvernon yes

Reply:
@aanm OK, will have a follow-up commit coming shortly once I get the build to pass without VM image downloads failing